src/App.css

```diff
@@ -10,13 +40,91 @@
   display: flex;
   flex-direction: column;
   font-size: 1em;
+  background-color: #dff2f2;
+  max-width: 450px;
```

Comment:
You may end up making this a little wider. The other team added an `item details` link/button to the shopping details page.